PR for 6.0.4

TomJaeger opened the pull request:
PR for 6.dev into 6.x for the 6.0.4 release

intoeetive committed:
move webfonts folder; update debugger styles on front-end;
resolves #911;
EECORE-1023

intoeetive committed:
fix #173 where live preview was not triggered when changing order or related enries
EECORE-1025

intoeetive committed:
improve UI for relationship fields in Grid
EECORE-1024

intoeetive committed:
fix #233 where Template Route could be accidently erased when editing template
EECORE-1030

intoeetive committed:
fix #347 where radio field label and values were swapped in channel form
EECORE-1031

Commit:
updated package.json for building styles

obfuscode committed:
Merge pull request #944 from ExpressionEngine/bug/fix-paths-packagejson

updated package.json for building styles

intoeetive committed:
fix #937 when MySQL error was shown after adding new folder in template files directory
EECORE-1064

Commit:
removed hardcoded height for list-item rows and adjusted line-height and spacing to allow for longer channel names

Jordan-Ellis committed:
Fixed a bug where a file field within a fluid field would populate all other file fields when selecting images

dougblackjr committed:
Adding check for last day of month

dougblackjr committed:
Changelog

intoeetive committed:
Fixed #908 where non-superadmin could not switch MSM sites
EECORE-1086

intoeetive committed:
display proper alerts if entry could not be deleted; fixes #941;
EECORE-1079

intoeetive committed:
fix #936 where some data were erased when saving member role
EECORE-1082

intoeetive committed:
fixed issue when roles list did not properly save for some menu sets

intoeetive committed:
fix #207 where file picker in textarea toolbar was not initialized for new fields in Fluid;
EECORE-1076

intoeetive committed:
fix #998 where exp:member:edit_profile was throwing an error when logged out;
EECORE-1093

intoeetive committed:
reverse commit [5e49311] because it breakes settings for compatible fieldtypes; fixes #995;
EECORE-1094

intoeetive committed:
add some waiting on tests

intoeetive committed:
fix issue when new post could not be published using Metaweblog API; EECORE-878

robinsowell committed:
Fixed a bug where the initial sorting of content when populating a custom field based on other fields was incorrect.

intoeetive committed:
fix #1020 where Fluid field dropdown could not accomodate very long list of fields

EECORE-1134

intoeetive committed:
disable horizontal resizing of textareas as that was causing issues in Chrome; fix #1022

EECORE-1133

intoeetive committed:
feature: pre-release script

intoeetive committed:
fix the quotes in script

intoeetive committed:
reorganize the tests

intoeetive committed:
update test scripts

intoeetive committed:
reorganize the tests

intoeetive committed:
reorganize cypress tests; add matchers to phpunit tests

intoeetive committed:
avoid setting HTTP header with CLI requests or PHPUnit tests

intoeetive committed:
update tests to be compatible with latest Cypress

intoeetive committed:
fix typo in test config

intoeetive committed:
use repo's own build.json for release

intoeetive committed:
fix yaml syntax

intoeetive committed:
run tests after build

intoeetive committed:
trigger tests from release

intoeetive committed:
workflow test

intoeetive committed:
update testing and releasing workflow

intoeetive committed:
fix z-index on 'add field' menu for Fluid fields

fixes #613
EECORE-1144

intoeetive committed:
fix wrapping of long field names in Fluid 'add field' dropdown menus

intoeetive committed:
add icons in Fluid 'add field' menus

intoeetive committed:
fix padding in dropdowns in File field inside Grid

intoeetive committed:
fix #1051 where order of quicklinks was not calculated correctly causing PHP error upon save

EECORE-1150

intoeetive committed:
apply search and sort parameters for Grid fields in LivePreview mode

fixes #1044
EECORE-1152

intoeetive committed:
bring back `comment_entries_tagdata` hook that was prematurely removed

TomJaeger committed:
Merge pull request #999 from ExpressionEngine/bug/6.x/member-profile-edit-tag

fix #998 where exp:member:edit_profile was throwing an error

TomJaeger committed:
Merge pull request #994 from ExpressionEngine/feature/6.x/permissions-check-for-entry-delete

display proper alerts if entry could not be deleted

TomJaeger committed:
Merge pull request #992 from ExpressionEngine/bug/6.x/msm-toggle

Fixed #908 where non-superadmin could not switch MSM sites

intoeetive committed:
update release process

intoeetive committed:
indicate prerelease

intoeetive committed:
make DP on pre-release

intoeetive committed:
increment DP version

intoeetive committed:
increase DP version

intoeetive committed:
update key name

intoeetive committed:
update the workflow to cancel what's previously running

intoeetive committed:
cancel runs differently

intoeetive committed:
cancel duplicates on bigger workflow runs as well

intoeetive committed:
yet another approach to cancel

intoeetive committed:
create tag with the release

intoeetive committed:
skip docs build

intoeetive committed:
fix action name

intoeetive committed:
use different action for release

intoeetive committed:
add token

intoeetive committed:
create tag from latest commit

intoeetive committed:
fix tests params

intoeetive committed:
create the tag before release

intoeetive committed:
tagging test

TomJaeger committed:
Merge pull request #993 from ExpressionEngine/bug/6.x/roles-and-menu-manager

fix #936 where some data were erased when saving member role

TomJaeger committed:
Merge pull request #980 from ExpressionEngine/feature/6.x/template-routes

fixes #233 where Template Route could be accidently erased

TomJaeger committed:
Merge pull request #964 from dougblackjr/fix/910-date-picker

Fix/910 date picker

TomJaeger committed:
Merge pull request #956 from Jordan-Ellis/bug/fluid-field-image-duplication

Fixed a bug where a file field within a fluid field would populate all other file fields when selecting images (Fixes #922)

TomJaeger committed:
Merge pull request #953 from ExpressionEngine/bug/eecore-1074-adjust-sidebar-spacing

EECORE-1074 removed hardcoded height for list-item rows and adjusted line-height

TomJaeger committed:
Merge branch '6.dev' into bug/6.x/field-populate-order

TomJaeger committed:
Merge pull request #1033 from ExpressionEngine/bug/6.x/field-populate-order

Fixed a bug where the initial sorting of content when populating a cu…

TomJaeger committed:
Merge pull request #1002 from ExpressionEngine/bug/6.x/metaweblog

fix issue when new post could not be published using Metaweblog API

intoeetive committed:
Merge branch '6.dev' into feature/6.x/filepicker-in-fluid

# Conflicts:
#	themes/ee/cp/css/common.min.css

intoeetive committed:
don't use draftt releases

TomJaeger committed:
Merge pull request #1054 from ExpressionEngine/feature/6.x/live-preview-and-grids

apply search and sort parameters for Grid fields in LivePreview mode

TomJaeger committed:
Merge pull request #1061 from ExpressionEngine/feature/6.x/saving-field-settings

saving field settings

TomJaeger committed:
Merge pull request #1062 from ExpressionEngine/feature/6.x/comments-extension-hooks

bring back `comment_entries_tagdata` hook that was prematurely removed

TomJaeger committed:
Merge pull request #1000 from ExpressionEngine/feature/6.x/filepicker-in-fluid

filepicker in Fluid textareas

TomJaeger committed:
Merge branch '6.dev' into feature/6.x/debugger-styles

TomJaeger committed:
Merge pull request #975 from ExpressionEngine/feature/6.x/debugger-styles

debugger styles

intoeetive committed:
update EE version to 6.0.4

intoeetive committed:
update build to 6.0.4

Commit:
EECORE-1159 removed export template option Fixes #1068

TomJaeger committed:
Merge pull request #1069 from ExpressionEngine/update/remove-export-template-option

EECORE-1159 removed export template option Fixes #1068

intoeetive committed:
update copyright date to 2021

TomJaeger committed:
Merge pull request #986 from ExpressionEngine/bug/6.x/fix-creating-template-group-from-files

fix for automatic creation of template groups from files

TomJaeger committed:
Merge pull request #981 from ExpressionEngine/feature/6.x/radio-field

fixes #347 where radio field label and values were swapped in channel form

TomJaeger committed:
Merge branch '6.dev' into feature/6.x/relationship-fields

TomJaeger committed:
Merge pull request #976 from ExpressionEngine/feature/6.x/relationship-fields

Merge pull request #907 from ExpressionEngine/6.dev

intoeetive committed:
Merge branch '6.dev' into feature/6.x/release-automation

# Conflicts:
#	tests/cypress/cypress/integration/settings/urls.ee6.js

intoeetive committed:
gitignore phpunit cache files

intoeetive committed:
update gulpfile

TomJaeger committed:
Merge pull request #1043 from ExpressionEngine/feature/6.x/release-automation

release automation

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/signed)](https://cla-assistant.io/ExpressionEngine/ExpressionEngine?pullRequest=1072) <br/>All committers have signed the CLA.

Commit:
EECORE-1036 fixes #922

intoeetive committed:
Merge pull request #1075 from ExpressionEngine/fix/EECORE-1036

EECORE-1036 fixes #922

intoeetive committed:
only save current fieldtype setting when swiching fieldtype;

fixes #995

Improves https://github.com/ExpressionEngine/ExpressionEngine/commit/5e493118feab3a5d12c6254874c85fbbe0225c3d

EECORE-1094

TomJaeger committed:
Merge pull request #1081 from ExpressionEngine/feature/6.x/saving-field-settings

only save current fieldtype setting when swiching fieldtype
